androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/RatingBar.kt

```diff
@@ -0,0 +1,65 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.toEmojis
+
+private const val SelectedEmojiAlpha = 1f
+private const val UnselectedEmojiAlpha = 0.5f
+
+@Composable
+fun RatingBar(
```

Comment:
This fun should not be in this PR

Reply:
I think because of conflict. Please rebase

Reply:
@suho fixed.